tensorboard/webapp/notification_center/_views/notification_center_component.ts

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Notification} from './view_types';
+
+@Component({
+  selector: 'notification-center-component',
+  templateUrl: './notification_center_component.ng.html',
+  styleUrls: ['./notification_center_component.css'],
+})
+export class NotificationCenterComponent {
+  @Input() notifications!: Notification[];
+  @Output() onNotificationChanged = new EventEmitter<Boolean>();
+
+  constructor() {}
+
+  onIconClicked() {
+    this.onNotificationChanged.emit();
+  }
+
+  onNotificationClicked(event: Event) {
+    event.stopPropagation();
```

Comment:
Hm, it's not clear to me from this PR alone why `stopPropagation` is needed.  If it is required to fix a bug that only appears in the followup PRs, could we move this change to one of the followups?

Reply:
Removed. Leave all functionality part to follow-up pr